app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,14 @@
+class LikesController < ApplicationController
+  def create
+    @post = Post.find(params[:id])
+    @like = Like.create(user: current_user, post: @post)
+
+    if @like.persisted?
+      flash[:success] = 'Liked successfully!'
+      redirect_to user_post_path(current_user, @post)
+    else
+      flash[:alert] = 'Something went wrong'
+      render :new
+    end
+  end
```

Comment:
- [ ] Great job :+1: for adding the create action here but unfortunately I'm unable to like a post. When I click on the like button, it lead me to a comment form. I'll kindly advise you to make sure  a user can actually like post :+1: